backfill.sql

```diff
@@ -88,21 +88,23 @@ BEGIN
     SELECT split_part(extversion, '.', 1)::INT INTO version FROM pg_catalog.pg_extension WHERE extname='timescaledb' LIMIT 1;
 
     IF version = 1 THEN
-        SELECT job_id INTO compression_job_id FROM _timescaledb_config.bgw_policy_compress_chunks b WHERE b.hypertable_id = move_compression_job.hypertable_id; 
+        SELECT job_id INTO compression_job_id FROM _timescaledb_config.bgw_policy_compress_chunks b WHERE b.hypertable_id = move_compression_job.hypertable_id;
     ELSE
-        SELECT s.job_id INTO compression_job_id FROM timescaledb_information.jobs j
-          INNER JOIN timescaledb_information.job_stats s ON j.job_id = s.job_id
-          WHERE j.proc_name = 'policy_compression' AND s.hypertable_schema = schema_name AND s.hypertable_name = table_name;
+        SELECT j.job_id INTO compression_job_id FROM timescaledb_information.jobs j
+          WHERE j.proc_name = 'policy_compression' AND j.hypertable_schema = schema_name AND j.hypertable_name = table_name;
+
     END IF;
 
-    IF compression_job_id IS NULL THEN 
+    IF compression_job_id IS NULL THEN
+        RAISE NOTICE 'No compression job found, no schedule to alter';
         old_time = NULL::timestamptz;
     ELSE
         SELECT next_start INTO old_time FROM _timescaledb_internal.bgw_job_stat WHERE job_id = compression_job_id FOR UPDATE;
 
+        RAISE NOTICE 'Altering compression job schedule: id=%, old_time=%, new_time%', compression_job_id, old_time, new_time;
         IF version = 1 THEN
             PERFORM alter_job_schedule(compression_job_id, next_start=> new_time);
-        ELSE 
+        ELSE
```

Comment:
Ditto, please revert:
```suggestion
        ELSE 
```